### DISCUSSION TITLE
How to use Excalidraw show grid mode

### DISCUSSION THREAD

**Discussion by NazarDoeH** [TriliumNext/Trilium#4710] [Q&A]
In Excalidraw, the grid can be enabled using Ctrl + ' keys. However, for some reason, it does not work for me. ( •_•)

**Comment by zadam** [comment]
Possibly because Trilium doesn't have the latest version yet. Upgrade is planned for next release.

**Reply by NazarDoeH** [reply]
Maybe.But I found that it's possible to turn on the grid. For that, I just set the variable 'gridModeEnabled' to true, and it works as expected. I think the problem might be with the keybind Ctrl + ', which doesn't work for some reason.

**Reply by jptrzy** [reply]
@NazarDoeH Where exactly did you enable this **gridModeEnabled** variable.

**Reply by jptrzy** [reply]
It looks like the only option for now is to edit trilium code while it is initializing [canvas widget](https://github.com/zadam/trilium/blob/82a437f2a83fc10299f3205713f7dcb04e2d047c/src/public/app/widgets/type_widgets/canvas.js#L443)

**Comment by pythontiger** [comment]
Has this been implemented already? I've got a Trilium instance up and running, but I'm struggling to find how to enable the grid. I can see the shortcut in the shortcut help menu, but it doesn't seem to do anything. It's also not showing up in the context menu, where I can find it in the official Excalidraw version. It'd be great if this could be added. And if this isn't planned, where can I set the variable gridModeEnabled? I haven't found the right location for doing this.
